### ISSUE TITLE
/* */ Comments in source Sass files are not closed correctly

### ISSUE THREAD

**Issue by liamprescott** [sass/node-sass#900]
/* */ Comment statements in source Sass files are not closed correctly in the output CSS when output-style is either 'compact' or 'compressed'

OS: Windows

CLI:
```
node-sass style/scss/main.scss style/css/main.css --source-map true --output-style compact
```

The following Sass:
```
/**
 * Comment
 */
.foo
{
  display:block;
} 
```

Outputs the following CSS:
```
/** This is global comment before a class statement /
.foo { display: block; }

/*# sourceMappingURL=main.css.map */
```
Please note the missing * before the closing / of the comment in the CSS.

**Comment by liamprescott** [comment]
Apologies. Output CSS should read:
```
/** Comment /
.foo { display: block; }

/*# sourceMappingURL=main.css.map */
```

**Comment by xzyfer** [comment]
This has been fixed in Libsass 3.2.0 (https://github.com/sass/libsass/pull/1137) and will be node-sass@3.0.0

**Comment by liamprescott** [comment]
Brilliant - any idea when node-sass@3.0.0 will be available?

**Comment by xzyfer** [comment]
Ideally this week. There are some outstanding PRs and issues that need to be resolved first.

**Comment by xzyfer** [comment]
Fixed in 3.0.0.
